generators/server/templates/_pom.xml

```diff
@@ -1200,7 +1209,7 @@
                 <!-- log configuration -->
                 <logback.loglevel>INFO</logback.loglevel>
                 <!-- default Spring profiles -->
-                <spring.profiles.active>prod</spring.profiles.active>
+                <spring.profiles.active>prod${profile.no-liquibase}</spring.profiles.active>
```

Comment:
Yes, you can Im currently testing your PR

Thanks & Regards,
Deepu

On Fri, May 20, 2016 at 1:27 PM, Erik Kemperman <notifications@github.com>
wrote:

> In generators/server/templates/_pom.xml
> <https://github.com/jhipster/generator-jhipster/pull/3620#discussion_r63992713>
> :
>
> > @@ -1200,7 +1209,7 @@
> >                  <!-- log configuration -->
> >                  <logback.loglevel>INFO</logback.loglevel>
> >                  <!-- default Spring profiles -->
> > -                <spring.profiles.active>prod</spring.profiles.active>
> > +                <spring.profiles.active>prod${profile.no-liquibase}</spring.profiles.active>
>
> Ah! Good call! I read in the docs that no-swagger has no effect in prod,
> so I excluded it here. But now I see that gradle also includes it for prod.
>
> I will add it when I get to campus. When I adapt the documentation, should
> I mention this option there as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3620/files/bae620709f7353d9bfb1e288e6159fc7acda9d3b#r63992713>
>